cmd/manager/scap.go

```diff
@@ -63,6 +66,17 @@ type resourceFetcherClients struct {
 	scheme    *runtime.Scheme
 }
 
+type roleNodes struct {
```

Comment:
Wouldn't this be more go-like to use a map of string-to a slice of strings?

Reply:
thanks for the advice, just fixed that